onadata/libs/utils/presigned_download_url.py

```diff
@@ -0,0 +1,44 @@
+import boto3
+from botocore.exceptions import ClientError
+import logging
+
+from django.conf import settings
+from django.core.files.storage import get_storage_class
+from django.http import HttpResponse, HttpResponseRedirect
+
+from wsgiref.util import FileWrapper
+
+
+def generate_media_download_url(file_path: str, expiration: int = 3600):
+    default_storage = get_storage_class()()
+    filename = file_path.split("/")[-1]
+    s3 = get_storage_class('storages.backends.s3boto3.S3Boto3Storage')()
+
+    if default_storage.__class__ != s3.__class__:
```

Comment:
I'm wondering whether this comparison is possible to do in a different way.  Feels weird to me to use `.__class__` for the comparison.  Maybe something like `type(default_storage) is type(s3)` could work?  idk